external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -258,6 +263,30 @@ impl<Progress: Write> DependencyGraphBuilder<Progress> {
             })
             .unwrap_or(None);
 
+        let root_pkg_id = custom_resolve_pkg_id(&root_manifest).with_context(|| {
+            format!(
+                "Resolving package name for '{}'",
+                root_manifest.package.name
+            )
+        })?;
+
+        let root_pkg_name = root_manifest.package.name;
+        if self.force_lock_file {
+            if let Some((_, _, Some(lock_string))) = digest_and_lock_contents {
+                eprintln!("Force-reading from lock file");
```

Comment:
This message maybe could be better, although you'll only see it if you ask for `--force-lock-file` so maybe it won't be so confusing